pkg/terminal/operator.go

```diff
@@ -2,17 +2,18 @@ package terminal
 
 import (
 	"context"
-
+	goErrors "errors"
```

Comment:
nitpick: I would better rename add alias for `k8sErrors` and let go error be `errors` )